autoload/ale/linter.vim

```diff
@@ -22,6 +22,7 @@ let s:default_ale_linter_aliases = {
 " Only cargo is enabled for Rust by default.
 let s:default_ale_linters = {
 \   'csh': ['shell'],
+\   'help': [],
```

Comment:
People might find it excessive that :help windows are suddenly getting linted. Sure, a small fraction have proselint installed, but it is possible that more linters will be added for this filetype.